app-tracking-protection/vpn-impl/src/main/res/layout/activity_tracking_protection_exclusion_list.xml

```diff
@@ -56,6 +56,16 @@
                     app:panelDrawable="@drawable/ic_alert_color_24"
                     android:layout_margin="16dp"/>
 
+            <TextView
+                android:id="@+id/excludedAppsFilterText"
```

Comment:
this needs a  `android:background="?selectableItemBackground"` so there's touch feedback

Reply:
I think we can keep this invisible until the first `ViewState` is received. Otherwise we can get to the situation where the drawable is visible but there's no text in the `TextView`

![Screenshot_20221018_141431](https://user-images.githubusercontent.com/531613/196430227-b6fb1f10-6d8d-4367-8c25-e24052b92000.png)

Reply:
- Added `android:background="?selectableItemBackground"`.
- `I think we can keep this invisible until the first ViewState is received` - this is done in the latest code (will be available in Part 5) along with loading states and empty banner.